refextract/references/kbs/publishers.kb

```diff
@@ -3,7 +3,7 @@ ELSEVIER---Elsevier
 NORTH HOLLAND---North-Holland
 BRANS---Brans
 BIRKHAEUSER---Birkhaeuser
-FERMILAB---Fermilab
+#FERMILAB---Fermilab
```

Comment:
so if Fermilab is not a publisher, it should be removed altogether
commenting out the line without further explanation isn't helpful
in particular since the list isn't alphabetically sorted